h5p-bildetema/src/hooks/useSiteLanguage.ts

```diff
@@ -1,5 +1,19 @@
+import { useLocation } from "react-router-dom";
+import { attributeLanguages } from "../../../common/constants/languages";
+import { LanguageCode } from "../../../common/types/LanguageCode";
 import { useL10n } from "./useL10n";
 
 export const useSiteLanguage = (): string => {
   return useL10n("htmlLanguageCode");
 };
+
+export const useCurrentLanguage = (): string => {
```

Comment:
Would be nice to move this into its own file

Reply:
Done ✔️